Source/Core/Core/CMakeLists.txt

```diff
@@ -505,6 +505,8 @@ add_library(core
   SysConf.h
   System.cpp
   System.h
+	TimePlayed.cpp
+	TimePlayed.h
```

Comment:
I think those are 1-tab indented instead of 2-space; can you make those consistent with the rest of the file?